bank.cc

```diff
@@ -0,0 +1,26 @@
+#include "bank.h"
+
+Bank::Bank( unsigned int numStudents ) : numStudents(numStudents) {
+	accounts = new int[numStudents];
+	for (unsigned int i = 0; i < numStudents; i ++) {
+		accounts[i] = 0;
+	}
+}
+
+void Bank::deposit( unsigned int id, unsigned int amount ) {
+	accounts[id] += amount;
+}
+
+void Bank::withdraw( unsigned int id, unsigned int amount ) {
+	while (accounts[id] < amount) {
```

Comment:
since withdraw is mutex... does this mean that nobody else can withdraw until this account gets enough money?

Reply:
wait no _Accept releases the mutex right